lib/spack/spack/test/optional_deps.py

```diff
@@ -96,6 +98,8 @@ def test_normalize(spec_and_expected, config, mock_packages):
     assert spec.eq_dag(expected, deptypes=False)
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
 def test_default_variant(config, mock_packages):
```

Comment:
Does this still fail? I think we'll want to get it working in this PR.

Reply:
This module is now passing.